selfdrive/controls/controlsd.py

```diff
@@ -208,6 +208,9 @@ def update_events(self, CS):
       self.events.add(EventName.vehicleModelInvalid)
     if not self.sm['liveLocationKalman'].posenetOK:
       self.events.add(EventName.posenetInvalid)
+    if not self.sm['frame'].recoverState < 2:
```

Comment:
use capnpn enum for the field

Reply:
this is more like just a counter number other than having any discrete meaning. I would say just keep it this way?